backend/src/govsg/middlewares/govsg-verification.middleware.ts

```diff
@@ -128,3 +128,25 @@ export const resendPasscodeCreationMessage = async (
     passcode_creation_wamid: passcodeCreationWamid,
   })
 }
+
+export const trackPasscodeReveal = async (
+  req: Request,
+  res: Response
+): Promise<Response | void> => {
+  const { govsg_message_id: govsgMessageId } = req.body
+  const userClickedAt = new Date()
```

Comment:
In `govsg-callback.service`, we [previous converted the date to milliseconds](https://github.com/opengovsg/postmangovsg/blob/9f78ca3ab2fad6084ff9e5a0ffad47aca1456eba/backend/src/govsg/services/govsg-callback.service.ts#L389). Any reason we don't do the same here?

Reply:
Because Meta returns seconds from epoch (sporadically mentioned in docs) and Date constructor returns milliseconds from epoch ([MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)).

This means we don't need any conversion here.